x/leverage/keeper/reserves_test.go

```diff
@@ -0,0 +1,76 @@
+package keeper_test
+
```

Comment:
This file (and the other completely new files, except `suite_test.go`) contains only functions which have been relocated from `keeper_test.go`